package.json

```diff
@@ -5,9 +5,7 @@
   "license": "MIT",
   "scripts": {
     "link": "link-parent-bin -s true && link-parent-bin -s true -c babel && link-parent-bin -s true -c demos && link-parent-bin -s true -c website",
```

Comment:
Do we still need "link"? At the moment you use it in `.travis.yml` only. But seems redundant.